test/lib/buildWebpackEntries.test.js

```diff
@@ -90,7 +93,8 @@ describe("src/lib/buildWebpackEntries", () => {
         index: indexPath,
         name: "used",
         reducers: reducersPath,
-        routes: routesPath
+        routes: routesPath,
+        group: null
```

Comment:
A part from the fixes on this test, you should test all the new functionality you add in `buildWebpackEntries.js`.